app/build.gradle

```diff
@@ -52,4 +52,6 @@ dependencies {
 
 // optional - Kotlin Extensions and Coroutines support for Room
     implementation "androidx.room:room-ktx:2.4.3"
+    implementation 'androidx.cardview:cardview:1.0.0'
+    implementation "androidx.cardview:cardview:1.0.0"
```

Comment:
Material Design also provides a MaterialCardView. Use that.

Reply:
No need to add these dependencies.

Reply:
Alright